src/ILCompiler.Compiler/src/IL/ILImporter.Scanner.cs

```diff
@@ -34,10 +34,15 @@ partial class ILImporter
         private class BasicBlock
         {
             // Common fields
+            public enum ImportState : int
+            {
+                Unmarked = 0, IsPending = -1
```

Comment:
Could you please add each value on its own line?

Also, it is not necessary to provide the integer values explicitly - we do not depend on them anywhere.